pytorch_lightning/trainer/training_loop.py

```diff
@@ -148,6 +146,9 @@ def on_train_end(self):
         # give accelerators a chance to finish
         self.trainer.accelerator.on_train_end()
 
+        # reset bookkeeping
+        self.trainer._running_stage = None
```

Comment:
Should we reset the model too ?

Reply:
Might be nice to have a NONE or NON_INITIALIZED running_stage. Any thoughts ?

Reply:
i'll leave that to https://github.com/PyTorchLightning/pytorch-lightning/issues/6095 if it's ever implemented